src/overlays/actors/ovl_En_Hint_Skb/z_en_hint_skb.c

```diff
@@ -97,96 +125,780 @@ static DamageTable D_80C21CE4 = {
+
+    if ((this->actionFunc != func_80C2016C) && (this->actionFunc != func_80C20590) &&
+        (this->actionFunc != func_80C2069C)) {
+        SkelAnime_Update(&this->skelAnime);
+    }
+
+    Actor_MoveWithGravity(&this->actor);
+    Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 15.0f, 30.0f, 60.0f, 0x1D);
+    func_80C21250(this, globalCtx);
+    func_80C20D64(this, globalCtx);
+    func_80C21320(this, globalCtx);
+}
+
+s32 func_80C21858(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnHintSkb* this = THIS;
+    f32 temp_f10;
+
+    if (limbIndex == 11) {
+        OPEN_DISPS(globalCtx->state.gfxCtx);
+
+        temp_f10 = fabsf(Math_SinS(globalCtx->state.frames * 6000) * 95.0f) + 160.0f;
+
+        gDPPipeSync(POLY_OPA_DISP++);
+        gDPSetEnvColor(POLY_OPA_DISP++, (s16)temp_f10, (s16)temp_f10, (s16)temp_f10, 255);
+
+        CLOSE_DISPS(globalCtx->state.gfxCtx);
+    } else if (limbIndex == 10) {
+        Matrix_GetStateTranslation(&this->actor.focus.pos);
+    } else if ((limbIndex == 12) && (this->unk_3DE == 1)) {
+        Matrix_InsertZRotation_s(0x71C, 1);
+    }
+
+    if (((limbIndex == 11) || (limbIndex == 12)) && (this->unk_3E8 & 2)) {
+        *dList = NULL;
+    }
+
+    return false;
+}
+
+void func_80C219D4(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    static Vec3f D_80C21E70 = { 800.0f, 1200.0f, 0.0f };
+    EnHintSkb* this = THIS;
+
+    if (!(this->unk_3E8 & 8)) {
+        Collider_UpdateSpheres(limbIndex, &this->collider);
+
+        if ((limbIndex == 11) && (this->unk_3E8 & 1) && !(this->unk_3E8 & 2)) {
+            Actor_SpawnBodyParts(&this->actor, globalCtx, 1, dList);
+            this->unk_3E8 |= 2;
+        } else if ((this->unk_3E8 & 4) && !(this->unk_3E8 & 8) && ((limbIndex != 11) || !(this->unk_3E8 & 1)) &&
+                   (limbIndex != 12)) {
+            Actor_SpawnBodyParts(&this->actor, globalCtx, 1, dList);
+        }
+
+        if (this->unk_3E2 != 0) {
+            if ((limbIndex == 2) || (limbIndex == 4) || (limbIndex == 5) || (limbIndex == 6) || (limbIndex == 7) ||
+                (limbIndex == 8) || (limbIndex == 9) || (limbIndex == 13) || (limbIndex == 14) || (limbIndex == 15) ||
+                (limbIndex == 16) || (limbIndex == 17) || (limbIndex == 18)) {
+                Matrix_GetStateTranslation(&this->unk_22C[this->unk_2DC]);
+                this->unk_2DC++;
+            } else if ((limbIndex == 11) && !(this->unk_3E8 & 2)) {
+                Matrix_MultiplyVector3fByState(&D_80C21E70, &this->unk_22C[this->unk_2DC]);
+                this->unk_2DC++;
+            }
+        }
+    }
+}
+
+void EnHintSkb_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    EnHintSkb* this = THIS;
+
+    this->unk_2DC = 0;
+    func_8012C28C(globalCtx->state.gfxCtx);
+    SkelAnime_DrawOpa(globalCtx, this->skelAnime.skeleton, this->skelAnime.jointTable, func_80C21858, func_80C219D4,
```

Comment:
Could you rename func_80C21858 to the overrideLimbDraw and func_80C219D4 to postLimbDraw?